augly/image/transforms.py

```diff
@@ -188,6 +188,52 @@ def apply_transform(
         return F.apply_pil_filter(image, filter_type=self.filter_type, metadata=metadata)
 
 
+class BarrelDistortion(BaseTransform):
+    def __init__(self, a: float = 0.0, b: float = 0.0, c: float = 0.0, d: float = 1.0, p: float = 1):
+        """
+        To see effects of the coefficients in detail refer to https://legacy.imagemagick.org/Usage/distorts/#barrel.
```

Comment:
Some of these lines look to be >90 cols. Please reformat so no line is >90 cols.

Reply:
I would leave formatting as I thought I would format the whole codebase on a different PR ? Should I format those files anyway ?

Reply:
Yes, please format the code you add (not the whole `transforms.py` file since that would introduce a lot of other changes which we want to keep separate). At least keep the lines <90 cols and things like that, and then we'll fix anything remaining in the formatting PR.